docs/source/examples/workspace/topologies/gce-net.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Did you forget to move/delete this

Reply:
I moved this but I forgot to delete it. Good catch

Reply:
the following is an example of libvirt topology making use of virt-customize (in comments)
Virt-customize command is used when the user would like to customize the libvirt image before it is provisioned.
refer : [virt-customize](http://libguestfs.org/virt-customize.1.html ) 
Note: User can use only one of the cloud-config types ie., cloud-init or virt-customize.
Most of the options are self-explanatory. Let me know if you need help with anything. 
```yaml
---
topology_name: libvirt-new
resource_groups:
  - resource_group_name: libvirt-new
    resource_group_type: libvirt
    resource_definitions:
      - role: libvirt_node
        name: centos71
        uri: qemu:///system
        count: 1
        #image_src: http://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-1608.qcow2.xz
        image_src: file:///home/srallaba/fedora_cb_28.qcow2
        memory: 1024
        vcpus: 1
        arch: x86_64
        networks:
          - name: default
        ssh_key: libvirt
        additional_storage: 10G
        cloud_config:
          virt_type: cloud-init
          users:
            - name: test
              gecos: test
              groups: wheel
              sudo: ALL=(ALL) NOPASSWD:ALL
            - name: someuser
              gecos: someuser
              groups: wheel
              sudo: ALL=(ALL) NOPASSWD:ALL
        #cloud_config:
        #  virt_type: virt-customize
        #  root_password: "testrootpwd" # insecure plaintext password
        #  run_script: "/home/srallaba/workspace/lp_ws/virt_customize/test.sh"
          #first_boot_commands: 
          #  - "echo \"this is created by the linchpin first_boot_commands\" >> first_boot_commands.txt"
        #  users:
        #    - name: test
        #      gecos: test
        #      groups: wheel
        #    - name: testuser2
        #      groups: wheel
        #      password: $1$ZA8hoL7w$eZexnQWoXTZajtfKGYy4f.  # encrypted password for "TestPWD"
        #    - name: someotheruser
        #      groups: wheel
        #      inject_ssh_keys: true  # by default linchpin generated libvirt key is injected.
        #    - name: srallaba
        #      groups: wheel
        #      inject_ssh_keys: true  # by default linchpin generated libvirt key is injected.
        #  packages:
        #    - gcc
        #    - openssh-server
        #  run_commands:
        #    - "echo \"hello world\""
        #    - "echo \"This is a file generated by run_commands parameters\" > /tmp/run_commands.txt"
```